### ISSUE TITLE
2 Suggestions from Discord 

### ISSUE THREAD

**Issue by LennartOschika** [gta-chaos-mod/ChaosModV#788]
From wohiwalabala on DIscord ( https://discord.com/channels/268280696601051136/268280696601051136/820320433340940308 )

1. Pacman (not the mission) - 3 star wanted level with a twist.

Basically, Pacman gives you 3 star wanted and spawns random yellow dots on the map. To decrease wanted level, you have to get a dot and "eat" (despawn) three cop cars by hitting them. 1 car = 1 star decreased. If any cop car hits the player before they reach a yellow dot, they get wasted.

2. Spawn Viewers - dunno if this is possible, but if controlling Chaos through twitch is, then this might be as well.

Basically, spawn as many people around the player as the number of Twitch viewers (or the ped limit). They all have the "You are famous" effect, and of the player kills any one of these peds, a random twitch viewer gets banned from the stream for 10 minutes.

**Comment by TheRealAyCe** [comment]
Put these in separate tickets please and close this one. Thanks :)
